core/src/test/java/org/apache/iceberg/util/TestJsonUtil.java

```diff
@@ -18,247 +18,202 @@
   }
 
   @Test
   public void getByteBufferOrNull() throws JsonProcessingException {
-    Assertions.assertThat(JsonUtil.getByteBufferOrNull("x", JsonUtil.mapper().readTree("{}")))
-        .isNull();
-    Assertions.assertThat(
-            JsonUtil.getByteBufferOrNull("x", JsonUtil.mapper().readTree("{\"x\": null}")))
+    assertThat(JsonUtil.getByteBufferOrNull("x", JsonUtil.mapper().readTree("{}"))).isNull();
+    assertThat(JsonUtil.getByteBufferOrNull("x", JsonUtil.mapper().readTree("{\"x\": null}")))
         .isNull();
 
     byte[] bytes = new byte[] {1, 2, 3, 4};
     String base16Str = BaseEncoding.base16().encode(bytes);
     String json = String.format("{\"x\": \"%s\"}", base16Str);
     ByteBuffer byteBuffer = JsonUtil.getByteBufferOrNull("x", JsonUtil.mapper().readTree(json));
-    Assertions.assertThat(byteBuffer.array()).isEqualTo(bytes);
+    assertThat(byteBuffer.array()).isEqualTo(bytes);
 
-    Assertions.assertThatThrownBy(
+    assertThatThrownBy(
             () -> JsonUtil.getByteBufferOrNull("x", JsonUtil.mapper().readTree("{\"x\": 23}")))
         .isInstanceOf(IllegalArgumentException.class)
         .hasMessage("Cannot parse byte buffer from non-text value: x: 23");
   }
 
   @Test
   public void getBool() throws JsonProcessingException {
-    Assertions.assertThatThrownBy(() -> JsonUtil.getBool("x", JsonUtil.mapper().readTree("{}")))
+    assertThatThrownBy(() -> JsonUtil.getBool("x", JsonUtil.mapper().readTree("{}")))
         .isInstanceOf(IllegalArgumentException.class)
         .hasMessage("Cannot parse missing boolean: x");
 
-    Assertions.assertThatThrownBy(
-            () -> JsonUtil.getBool("x", JsonUtil.mapper().readTree("{\"x\": null}")))
+    assertThatThrownBy(() -> JsonUtil.getBool("x", JsonUtil.mapper().readTree("{\"x\": null}")))
         .isInstanceOf(IllegalArgumentException.class)
         .hasMessage("Cannot parse to a boolean value: x: null");
 
-    Assertions.assertThatThrownBy(
-            () -> JsonUtil.getBool("x", JsonUtil.mapper().readTree("{\"x\": \"23\"}")))
+    assertThatThrownBy(() -> JsonUtil.getBool("x", JsonUtil.mapper().readTree("{\"x\": \"23\"}")))
         .isInstanceOf(IllegalArgumentException.class)
         .hasMessage("Cannot parse to a boolean value: x: \"23\"");
 
-    Assertions.assertThatThrownBy(
-            () -> JsonUtil.getBool("x", JsonUtil.mapper().readTree("{\"x\": \"true\"}")))
+    assertThatThrownBy(() -> JsonUtil.getBool("x", JsonUtil.mapper().readTree("{\"x\": \"true\"}")))
         .isInstanceOf(IllegalArgumentException.class)
         .hasMessage("Cannot parse to a boolean value: x: \"true\"");
 
-    Assertions.assertThat(JsonUtil.getBool("x", JsonUtil.mapper().readTree("{\"x\": true}")))
-        .isTrue();
-    Assertions.assertThat(JsonUtil.getBool("x", JsonUtil.mapper().readTree("{\"x\": false}")))
-        .isFalse();
-  }
-
-  @Test
-  public void getIntArrayOrNull() throws JsonProcessingException {
```

Comment:
this also needs to be reverted. In fact, can you please revert all changes to this file. I believe the only change that this file requires is switching from `org.junit.Test` to `org.junit.jupiter.api.Test` (everything else just adds too much visual noise